requirements-test.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
 pytest==6.2.4
 pytest-socket==0.4.0
-flake8==3.9.0
-pre-commit==2.11.1
+flake8==3.9.2
```

Comment:
We should migrate to pipenv, so much easier with [dev-packages[ and piplock...

Reply:
We could do - but lets leave that to another PR ;)

There is also https://python-poetry.org/

I've not used but looks quite nice.